src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -1613,6 +1632,9 @@ public void Close()
 
                 if (!attach)
```

Comment:
>attach [](start = 21, length = 6)

Should we be doing this someplace in attach as well?

Reply:
I don't think so because it looks like any requests happen after -target-attach is run, which means that the code is loaded in the final memory location already.